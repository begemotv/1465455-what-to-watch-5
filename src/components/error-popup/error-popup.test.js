import React from "react";
import renderer from "react-test-renderer";

import ErrorPopup from "./error-popup";

test(`ErrorPopup renders correctly`, () => {
  const tree = renderer
    .create(
        <ErrorPopup />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
