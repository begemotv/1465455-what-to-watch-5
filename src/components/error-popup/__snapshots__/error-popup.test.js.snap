// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPopup renders correctly 1`] = `
<div
  className="Toastify"
/>
`;
